### PR TITLE
Collecting periodic support bundle

### PR DESCRIPTION
**Pull request by ketanarlulkar**
# Problem Statement
-  Collecting support bundle
-  Execution Logs: [CORTX-28106](https://jts.seagate.com/browse/CORTX-28106?focusedCommentId=2323652&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2323652)

# Design
-  For Bug, Describe the fix here.
-  For new test addition post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-  [x] New/Affected tests are executed
-  [x] Attach test execution logs

# Review Checklist 
  Checklist for Author
-  [x] JIRA number/GitHub Issue added to PR
-  [x] PR is self reviewed
-  [x] Jira and state/status is updated and JIRA is updated with PR link
-  [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe

### CODE REVIEW

**Review by kulrk — APPROVED** [review]

File: corio.py [hunk 1]
```diff
@@ -272,6 +261,26 @@ def convert_size(size_bytes):
     return f"{size} {size_name[check_pow]}"
 
 
+def support_bundle_process(interval, sb_identifier):
+    """Support bundle wrapper."""
```
Comment (priyankaborawake) [inline corio.py]: Please add docstring

**Review by priyankaborawake — APPROVED** [review]

File: config/corio_config.yaml [hunk 2]
```diff
@@ -4,3 +4,4 @@ sb_interval_mins: 60
 max_no_of_sb: 5
 perform_health_check: True
 hc_interval_mins: 60
+nfs_server:
```
Comment (DhobaleVishal) [inline config/corio_config.yaml]: Need some server details if we are not keeping it here then better to pass it through argument.

File: src/commons/utils/support_bundle_utils.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    :param local_dir_path: local dir path on client
+    :param scripts_path: services scripts path on master node
+    :return: Boolean
+    """
+    host, user, passwd = None, None, None
+    for node in CLUSTER_CFG["nodes"]:
+        if node["node_type"] == "master":
+            host, user, passwd = node["hostname"], node["username"], node["password"]
+            break
+
```
Comment (DhobaleVishal) [inline src/commons/utils/support_bundle_utils.py]: remove empty lines from code.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]

File: corio.py [hunk 4]
```diff
@@ -107,8 +109,7 @@ def parse_args():
     return parser.parse_args()
```
Comment (DhobaleVishal) [inline corio.py:112]: sb collection should be conditional. Please add argument for same here.

**Comment by VikasKumarC** [comment]
Hi @ketanarlulkar  Check codacy issues

**Review by DhobaleVishal — APPROVED** [review]

**Pull request merged**